core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/exception/LimitExceededException.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.commonutil.exception;
+
+public class LimitExceededException extends RuntimeException {
```

Comment:
Could this be modified to extend GafferRuntimeException? This is required to integrate with the REST API.